client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -80,9 +79,22 @@ class DomainToPlanNudge extends Component {
 		} );
 
 		page( `/checkout/${ siteId }/personal` );
+	};
+
+	renderRegular() {
+		const { siteId, translate } = this.props;
+		return (
+			<UpgradeNudge
+				title={ translate( 'Upgrade to a Personal Plan and Save!' ) }
+				message={ translate( 'Get live chat support and no ads bundled with your domain!' ) }
```

Comment:
It currently barely fits in a mobile view. 😄 Alternatively I could try to fix the nudge to support 2 lines without distorting the svg logo.